tests/test_node_differ.py

```diff
@@ -328,8 +336,15 @@ def test_moved_section():
     diff = EditTypes(prev_wikitext, curr_wikitext, lang='en').get_diff()
     assert expected_changes == diff
 
-def test_insert_cjk_punctuations():
+def test_change_cjk_punctuations():
```

Comment:
just a note for later: this test actually is incorrect because it essentially is changing a period to a comma so it should lead one sentence changing and one being removed (because two sentences are being merged into one). that won't be detected properly though until we address Issue #51. this doesn't need to be fixed in this PR but let's address it as a high priority in the next PR

Reply:
Alright @geohci I would work on it.